backend/compact-connect/stacks/persistent_stack/__init__.py

```diff
@@ -81,13 +82,15 @@ def __init__(
             user_pool_email_settings = UserPoolEmail.with_cognito()
 
         staff_prefix = f'{app_name}-staff'
+
         self.staff_users = StaffUsers(
-            self, 'StaffUsers',
+            self, 'StaffUsersGreen',
```

Comment:
I had to rename the user pool to avoid resource naming conflicts when the deploy creates a new user pool with the current updates. 'Green' here is a reference to a kind of 'Blue/Green deploy' strategy.

Reply:
Got it thanks!